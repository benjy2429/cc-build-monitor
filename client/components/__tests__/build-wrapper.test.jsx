import React from 'react';
import { shallow } from 'enzyme';
import BuildWrapper from '../build-wrapper';

const defaultProps = {
  builds: [
    { repo: 'project1', buildNumber: 1 },
    { repo: 'project2', buildNumber: 2 },
  ],
};

describe('Build wrapper', () => {
  it('renders correctly', () => {
    const component = shallow(
      <BuildWrapper {...defaultProps} />,
    );

    expect(component).toMatchSnapshot();
  });
});
