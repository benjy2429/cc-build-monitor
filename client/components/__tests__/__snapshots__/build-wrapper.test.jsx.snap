// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Build wrapper renders correctly 1`] = `
<div
  className="build-wrapper"
>
  <Build
    buildNumber={1}
    repo="project1"
  />
  <Build
    buildNumber={2}
    repo="project2"
  />
</div>
`;
